plutus-ir/src/Language/PlutusIR.hs

```diff
@@ -186,3 +187,38 @@ instance (PLC.PrettyClassicBy configName (tyname a), PLC.PrettyClassicBy configN
 
 prettyDef :: (PLC.PrettyBy (PLC.PrettyConfigClassic PLC.PrettyConfigName) a) => a -> Doc ann
 prettyDef = prettyBy $ PLC.PrettyConfigClassic PLC.defPrettyConfigName
+
+-- See note [Default pretty instances for PLC]
+instance (PLC.PrettyBy (PLC.PrettyConfigClassic PLC.PrettyConfigName) (tyname a)) =>
```

Comment:
Is it possible to write `PLC.PrettyClassicBy PLC.PrettyConfigName` instead of `PLC.PrettyBy (PLC.PrettyConfigClassic PLC.PrettyConfigName)`?

Reply:
Ah yes, I'm a bit lost in the maze of type snynonyms.